solver/src/settlement.rs

```diff
@@ -11,14 +12,13 @@ use primitive_types::{H160, U256};
 use shared::conversions::U256Ext as _;
 use std::collections::HashMap;
 
-pub use settlement_encoder::SettlementEncoder;
-
 #[derive(Clone, Debug, Default, Eq, PartialEq)]
 pub struct Trade {
     pub order: Order,
     pub sell_token_index: usize,
     pub buy_token_index: usize,
     pub executed_amount: U256,
+    pub scaled_fee_amount: U256,
```

Comment:
I think a comment on what we mean by scaled here and in the other struct where it appears would be good.